### PR TITLE
Remove unused variables, no longer in the basket code.

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: mozmeao-fr/basket-prod/clock-deploy.yaml [hunk 1]
```diff
@@ -197,11 +169,6 @@ spec:
           value: basket-prod
         - name: SUBHUB_OPP_RECORD_TYPE
           value: 0124O000000R0ODQA0
-        - name: SUPERTOKEN
```
Comment (stevejalim) [inline mozmeao-fr/basket-prod/clock-deploy.yaml:200]: Thinking aloud: I see a ref to `supertoken` in `basket-client` https://github.com/mozilla/node-basket/blob/master/index.js#L61 -- but it doesn't look like it's deployed as part of main basket (which makes sense - it's a client, after all)

**Review by stevejalim — DISMISSED** [review]
This makes sense if I've correctly deduced that ExactTarget is what we used before it got bought by Salesforce, but am wondering where things like `SFDC_USERNAME` are coming from, as would make sense to cut those out too. What am I forgetting here?

File: mozmeao-fr/basket-prod/clock-deploy.yaml (hunk 1 above)
Comment (robhudson) [reply mozmeao-fr/basket-prod/clock-deploy.yaml:200]: Yeah it was removed awhile ago and returns a 404 now... https://github.com/mozmeao/basket/blob/master/basket/news/views.py#L715

**Comment by robhudson** [comment]
I was going to wait to remove the SFDC configs until after the code that references them is removed. I just happened to notice these older configs that could have been removed awhile ago and thought I'd do a quick clean-up.

**Review by pmac — APPROVED** [review]
Cleanup is always appreciated 🏅

**Comment by stevejalim** [comment]
Yep, thanks for the housekeeping @robhudson! 👍

**Review by stevejalim — APPROVED** [review]

**Comment by robhudson** [comment]
Thanks y'all!

**Pull request merged**